Assets/Global/Scripts/Player/Player.cs

```diff
@@ -17,13 +18,16 @@ public class Player : MonoBehaviour
 
     [Header("Attack")]
     public float attackResettingTime = 2f;
-    public float TailTurnSpeed = 40f;
+    public float tailTurnDuration = 0.1f;
     public int slamDamage = 10;
-    public int firstTailDamage = 10;
-    public int secondTailDamage = 15;
+    public int leftTailDamage = 10;
+    public int rightTailDamage = 15;
+
+    public int flipDamage = 20;
 
     public float slamForce = 2.0f;
-    public BoxCollider TransformTail;
+
+    public GameObject Tail;
```

Comment:
can this not be of type Tail ?
So in  Attackingstate.cs you dont have to Player.Tail.GetComponent<Tail>() to get the tail script?
Or is there a reason this is of type GameObject?